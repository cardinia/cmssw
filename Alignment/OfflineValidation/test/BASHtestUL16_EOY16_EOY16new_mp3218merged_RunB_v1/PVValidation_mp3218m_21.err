26-Sep-2019 12:49:22 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/84E501A8-C80B-E911-B3F9-AC1F6B1AF000.root
26-Sep-2019 12:49:27 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/84E501A8-C80B-E911-B3F9-AC1F6B1AF000.root
Begin processing the 1st record. Run 272786, Event 467202, LumiSection 1 on stream 0 at 26-Sep-2019 12:50:06.804 CEST
Begin processing the 1001st record. Run 272786, Event 3314614, LumiSection 5 on stream 0 at 26-Sep-2019 13:05:13.966 CEST
Begin processing the 2001st record. Run 272786, Event 7187712, LumiSection 10 on stream 0 at 26-Sep-2019 13:08:01.407 CEST
Begin processing the 3001st record. Run 272786, Event 10305846, LumiSection 14 on stream 0 at 26-Sep-2019 13:10:48.752 CEST
Begin processing the 4001st record. Run 272786, Event 13574820, LumiSection 19 on stream 0 at 26-Sep-2019 13:13:43.055 CEST
Begin processing the 5001st record. Run 272786, Event 17534339, LumiSection 24 on stream 0 at 26-Sep-2019 13:16:31.807 CEST
Begin processing the 6001st record. Run 272786, Event 20996940, LumiSection 29 on stream 0 at 26-Sep-2019 13:19:23.068 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:19:54 CEST  Run: 272786 Event: 21085512
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 7001st record. Run 272786, Event 24298588, LumiSection 33 on stream 0 at 26-Sep-2019 13:22:09.222 CEST
Begin processing the 8001st record. Run 272786, Event 27887947, LumiSection 38 on stream 0 at 26-Sep-2019 13:25:01.243 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:26:18 CEST  Run: 272786 Event: 28924523
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 2
%MSG
Begin processing the 9001st record. Run 272786, Event 31510478, LumiSection 43 on stream 0 at 26-Sep-2019 13:27:44.835 CEST
Begin processing the 10001st record. Run 272786, Event 34399510, LumiSection 47 on stream 0 at 26-Sep-2019 13:30:27.673 CEST
Begin processing the 11001st record. Run 272786, Event 38087871, LumiSection 52 on stream 0 at 26-Sep-2019 13:33:13.747 CEST
Begin processing the 12001st record. Run 272786, Event 40934666, LumiSection 56 on stream 0 at 26-Sep-2019 13:35:54.335 CEST
26-Sep-2019 13:37:42 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/84E501A8-C80B-E911-B3F9-AC1F6B1AF000.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       2        2
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          272786/21085512  272786/28924523  
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 2                   2
System                  3                   3

dropped waiting message count 0
[8.744MB/8.744MB][100%][==================================================][8.744MB/s]  [8.744MB/8.744MB][100%][==================================================][8.744MB/s]  
