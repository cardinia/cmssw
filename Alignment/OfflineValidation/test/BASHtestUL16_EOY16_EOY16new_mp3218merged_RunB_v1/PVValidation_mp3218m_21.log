000 (3189503.021.000) 09/26 11:21:20 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189503.021.000) 09/26 12:34:27 Started transferring input files
	Transferring to host: <188.185.165.227:9618?addrs=188.185.165.227-9618+[2001-1459-301-4b--100-1dd]-9618&noUDP&sock=4670_2ff7_99865>
...
040 (3189503.021.000) 09/26 12:34:27 Finished transferring input files
...
001 (3189503.021.000) 09/26 12:34:27 Job executing on host: <188.185.165.227:9618?addrs=188.185.165.227-9618+[2001-1459-301-4b--100-1dd]-9618&noUDP&sock=4283_5c38_3>
...
006 (3189503.021.000) 09/26 12:34:35 Image size of job updated: 9688
	10  -  MemoryUsage of job (MB)
	9688  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 12:39:35 Image size of job updated: 30464
	16  -  MemoryUsage of job (MB)
	15980  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 12:44:36 Image size of job updated: 51864
	33  -  MemoryUsage of job (MB)
	32960  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 12:49:37 Image size of job updated: 361772
	274  -  MemoryUsage of job (MB)
	280164  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 12:54:37 Image size of job updated: 567568
	444  -  MemoryUsage of job (MB)
	453884  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 12:59:38 Image size of job updated: 572532
	449  -  MemoryUsage of job (MB)
	458848  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 13:04:39 Image size of job updated: 657180
	528  -  MemoryUsage of job (MB)
	540148  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 13:09:39 Image size of job updated: 657844
	529  -  MemoryUsage of job (MB)
	540804  -  ResidentSetSize of job (KB)
...
006 (3189503.021.000) 09/26 13:14:39 Image size of job updated: 659176
	530  -  MemoryUsage of job (MB)
	542136  -  ResidentSetSize of job (KB)
...
040 (3189503.021.000) 09/26 13:38:32 Started transferring output files
...
040 (3189503.021.000) 09/26 13:38:32 Finished transferring output files
...
005 (3189503.021.000) 09/26 13:38:32 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:34:50, Sys 0 00:00:07  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:34:50, Sys 0 00:00:07  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4892  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	4892  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     8997        1   1458164 
	   Memory (MB)          :      530     2000      2000 
...
