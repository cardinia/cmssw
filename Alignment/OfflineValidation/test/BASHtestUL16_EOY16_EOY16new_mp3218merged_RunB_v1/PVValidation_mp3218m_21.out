Job started at  Thu Sep 26 12:34:27 CEST 2019
batch dir: /pool/condor/dir_25490 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_272786_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 8.8M -rw-r--r--. 1 tkello zh 8.8M Sep 26 13:38 PVValidation_mp3218m_272786.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:35 PVValidation_mp3218m_272786_cfg.py -rw-r--r--. 1 tkello zh 3.6K Sep 26 13:38 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:43 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 26 12:34 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:49 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:34 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:34 var
Job ended at  Thu Sep 26 13:38:32 CEST 2019
